frontend/src/scenes/onboarding/sdks/SDKs.tsx

```diff
@@ -1,54 +1,92 @@
-import { LemonDivider, LemonInput, LemonSelect } from '@posthog/lemon-ui'
+import { LemonButton, LemonDivider, LemonSelect } from '@posthog/lemon-ui'
 import { sdksLogic } from './sdksLogic'
-import { LemonCard } from 'lib/lemon-ui/LemonCard/LemonCard'
 import { useActions, useValues } from 'kea'
 import { OnboardingStep } from '../OnboardingStep'
 import { SDKSnippet } from './SDKSnippet'
 import { onboardingLogic } from '../onboardingLogic'
+import { useEffect } from 'react'
+import React from 'react'
+import { SDKInstructionsMap } from '~/types'
 
-export function SDKs({ usersAction }: { usersAction?: string }): JSX.Element {
-    const { setSourceFilter, setSelectedSDK } = useActions(sdksLogic)
+export function SDKs({
+    usersAction,
+    sdkInstructionMap,
+}: {
+    usersAction?: string
+    sdkInstructionMap: SDKInstructionsMap
+}): JSX.Element {
+    const { setSourceFilter, setSelectedSDK, setAvailableSDKInstructionsMap } = useActions(sdksLogic)
     const { sourceFilter, sdks, selectedSDK, sourceOptions } = useValues(sdksLogic)
     const { productKey } = useValues(onboardingLogic)
 
+    useEffect(() => {
+        setAvailableSDKInstructionsMap(sdkInstructionMap)
+    }, [])
+
     return (
         <OnboardingStep
             title={`Where are you ${usersAction || 'collecting data'} from?`}
-            subtitle="Pick one to start and add more sources later."
+            subtitle="Pick one or two to start and add more sources later."
         >
-            <div className="flex gap-x-4">
-                <LemonInput placeholder="Search for a source" type="search" />
-                {sourceOptions.length > 1 && (
-                    <LemonSelect
-                        allowClear
-                        onChange={(v) => setSourceFilter(v)}
-                        options={sourceOptions}
-                        placeholder="Select a source type"
-                        value={sourceFilter}
-                    />
+            <LemonDivider className="my-8" />
+            <div className="flex gap-x-8 mt-8">
+                <div className={`flex flex-col gap-y-2 flex-wrap gap-x-4 min-w-50`}>
+                    {sourceOptions.length > 1 && (
+                        <LemonSelect
+                            allowClear
+                            onChange={(v) => setSourceFilter(v)}
+                            options={sourceOptions}
+                            placeholder="Select a source type"
+                            value={sourceFilter}
+                            className="w-full"
+                        />
+                    )}
+                    {sdks?.map((sdk) => (
+                        <React.Fragment key={`sdk-${sdk.key}`}>
+                            {selectedSDK?.key == sdk.key ? (
+                                <LemonButton
```

Comment:
The icons on these buttons look off center. It's the same in the selected / unselected states

<img width="647" alt="Screenshot 2023-09-06 at 10 59 01" src="https://github.com/PostHog/posthog/assets/6685876/8bca2f2d-8400-48a3-96fd-5c875cbdd02d">

Reply:
Ben fixed this!